Source/Components/Touchpoints/Types/FlexCoverCarousel/Card/MLBusinessFlexCoverCarouselItemView.swift

```diff
@@ -69,6 +69,7 @@ public class MLBusinessFlexCoverCarouselItemView: UIView {
         label.textAlignment = .left
         label.textColor = MLStyleSheetManager.styleSheet.whiteColor
         label.translatesAutoresizingMaskIntoConstraints = false
+        label.isAccessibilityElement = false
```

Comment:
Porque marcamos todos estos labels como `isAccessibilityElement = false`

Reply:
creo que esta bien, si no lo dejas en false el coso de accesibilidad de ios intenta leer lo que tiene el text del label

Reply:
Porque de esa manera podemos tomar el control de que es lo que va a decir VoiceOver. En este caso cada cell de la collection es como "un gran botón". Todos los labels tienen accesibilidad por defecto, por eso se lo sacamos y el construimos uno nuevo a gusto y piacere para mejorarle la experiencia al user. De todas maneras, primero verificamos si BE nos manda esa descripción y en tal caso no usamos nada.